libraries/codegen-tooling/inc/can_msg_defs.h

```diff
@@ -20,7 +20,9 @@ typedef enum {
   SYSTEM_CAN_DEVICE_SOLAR_MASTER_REAR = 11,
   SYSTEM_CAN_DEVICE_SENSOR_BOARD = 12,
   SYSTEM_CAN_DEVICE_CHARGER = 13,
-  NUM_SYSTEM_CAN_DEVICES = 14
+  SYSTEM_CAN_DEVICE_STEERING_CONTROLS = 14,
```

Comment:
It looks like these changes were made manually. However, these files are autogenerated. Please look at codegen-tooling and open a review for these changes there as well.

Reply:
@mishazharov  hmm not sure what you mean, we added the new CAN device ID's and defined new CAN messages, how does it generate that automatically? maybe I'm missing something

Reply:
Everything in the codegen tooling folder is generated automatically by https://github.com/uw-midsun/codegen-tooling/

In order for codegen-tooling to automatically generate the appropriate CAN messages, it needs to be modified. Manual changes to these files are definitely no good

Reply:
okay gotcha, will fix ASAP